src/app/Media/IMediaSession.cs

```diff
@@ -137,11 +88,5 @@ public interface IMediaSession
         /// being sent for the tone.</param>
         Task SendDtmf(byte tone, CancellationToken ct);
 
-        /// <summary>
-        /// Closes the session. This will stop any audio/video capturing and rendering devices as
-        /// well as the RTP and RTCP sessions and sockets.
-        /// </summary>
-        /// <param name="reason">Optional. A descriptive reason for closing the session.</param>
-        void Close(string reason);
-    }
```

Comment:
It's a bit strange to have an interface inheriting from another interface? Is there a reason for it?

Reply:
see an example of MSRPSession class